.travis.yml

```diff
@@ -2,18 +2,15 @@ sudo: true
 dist: trusty
 language: node_js
 node_js:
-- '5'
+- '6'
 before_script:
-- npm install bower
+- npm install -g bower web-component-tester
 - bower install
-script: wct --plugin sauce
-cache:
-  directories:
-  - node_modules
+script: xvfb-run wct
```

Comment:
Hmm...it just means I don't think it'll run on sauce, which means it'll only test the browsers that's on the travis VM - just Chrome I think - but I'll have a dig into it, particularly if the Polymer guys aren't using sauce.